src/messages/javascript.js

```diff
@@ -126,6 +126,13 @@ export const UNSUPPORTED_API = {
   description: _('This API has not been implemented by Firefox.'),
 };
 
+export const PERMISSIONS_API = {
+  code: 'PERMISSIONS_API',
+  message: null,
+  messageFormat: _(' API has been used but '),
+  description: _('Used API without permission'),
+};
+
```

Comment:
I would specify and use this object similarly to `UNSUPPORTED_API` so that we can substitute the actual API and needed permission in the message and description.

Or use the function based approach similarly to `deprecatedApi`